mapie/regression.py

```diff
@@ -222,14 +223,46 @@ def _check_parameters(self) -> None:
                 "Allowed values are 'naive', 'base', 'plus' and 'minmax'."
             )
 
-        if self.agg_function not in self.valid_agg_functions_:
+        check_n_jobs(self.n_jobs)
+        check_verbose(self.verbose)
+
+    def _check_agg_function(
+        self, agg_function: Optional[str] = None
+    ) -> Optional[str]:
+        """
+        Check if ``agg_function`` is correct, and consistent with other
+        arguments.
+
+        Parameters
+        ----------
+        agg_function : Optional[str], optional
+            Aggregation function's name to check, by default ``None``.
+
+        Returns
+        -------
+        str
+            ``agg_function`` itself or ``"mean"``.
+
+        Raises
+        ------
+        ValueError
+            If ``agg_function`` is not in [``None``, ``"mean"``, ``"median"``],
+            or is ``None`` while cv class is ``Subsample``.
+        """
+        if agg_function not in self.valid_agg_functions_:
             raise ValueError(
                 "Invalid aggregation function "
                 "Allowed values are None, 'mean', 'median'."
             )
 
-        check_n_jobs(self.n_jobs)
-        check_verbose(self.verbose)
+        if isinstance(self.cv, Subsample) and (agg_function is None):
```

Comment:
I'm seeing this a bit too late, but why does the user now have to specify explicitely an `agg_function` now ? If I'm correct, when `agg_function` was not specified, a "mean" was used, why this change ?